src/android/GeofenceTransitionsService.java

```diff
@@ -0,0 +1,240 @@
+import android.graphics.BitmapFactory;
+import android.graphics.Color;
+import android.location.Location;
+import android.location.LocationListener;
+import android.location.LocationManager;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.os.IBinder;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.app.TaskStackBuilder;
+import android.text.TextUtils;
+import android.util.Log;
+
+import com.google.android.gms.location.Geofence;
+import com.google.android.gms.location.GeofencingEvent;
+import android.app.Service;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Listener for geofence transition changes.
+ *
+ * Receives geofence transition events from Location Services in the form of an Intent containing
+ * the transition type and geofence id(s) that triggered the transition. Creates a notification
+ * as the output.
+ */
+public class GeofenceTransitionsService extends Service {
+
+    protected static final String TAG = "GeofenceTransitionsIS";
+    private double lat, lon, alt;
+    private String timestamp = "yyyyMMdd'T'HH:mm:ss";
+
+    public DBHelper GPS_dbManager;
+    public Geofence geofence;
+    Intent intent;
+    LocationManager locationManager = null;
+    public GetLocationInfo listener;
+
+    private static final int LOCATION_INTERVAL = 10000;
+    private static final float LOCATION_DISTANCE = 10f;
+
+
+    private class GetLocationInfo implements LocationListener{
+        @Override
+        public void onLocationChanged(Location location) {
+            lat = location.getLatitude();
+            lon = location.getLongitude();
+            alt = location.getAltitude();
+
+            SimpleDateFormat GPStime = new SimpleDateFormat("yyyyMMdd'T'HH:mm:ss");
+            timestamp = GPStime.format (location.getTime());
+        }
+
+        @Override
+        public void onStatusChanged(String s, int i, Bundle bundle) {}
+
+        @Override
+        public void onProviderEnabled(String s) {}
+
+        @Override
+        public void onProviderDisabled(String s) {}
+    }
+
+    private void initializeLocationManager() {
+        locationManager = (LocationManager) getSystemService(Context.LOCATION_SERVICE);
+        listener = new GetLocationInfo();
+        locationManager.requestLocationUpdates(LocationManager.NETWORK_PROVIDER, LOCATION_INTERVAL, LOCATION_DISTANCE, listener);
+        locationManager.requestLocationUpdates(LocationManager.GPS_PROVIDER, LOCATION_INTERVAL, LOCATION_DISTANCE, listener);
+
+    }
+
+
+    @Override
+    public void onCreate(){ super.onCreate(); intent = new Intent(TAG); }
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+        initializeLocationManager();
+
+        GeofencingEvent geofencingEvent = GeofencingEvent.fromIntent(intent);
```

Comment:
this doesn't look like it's tested at least not thoroughly :-1: . For some reason you keep mixing up geofencing and location updates. Again, these are not the same thing:

* location updates are sent _every time_ the phone registers a new location. for this you need the LocationManager, which needs to be called from a Service that keeps running in the background because the LocationManager needs a live LocationListener to receive updates. this is _independent of geofencing_
* geofences are only sent when a defined region boundary is crossed. For this you need a IntentService because updates are delivered as intents. 

You have implemented it somewhat the other way around, why? 

* MultiRegionService is now an IntentService, but it is started as a service. If this works at all, it means the MultiRegionService dies immediately after it has finished its `onHandleIntent` method, while it should keep running to receive LocationListener callbacks.
* GeofenceTransitionService, which should be an IntentService, is now a Service. This means GeofenceTransitionService will keep running while  it should really just handle geofencing updates and then die.

While I doubt this will work (at least reliably) in the first place, what's worse the **original problem is still not solved**: location updates are only started when the first geofence transition has occured, which may be never. That's wrong. Location updates shall be received independent of whether the user has crossed a geofence boundary.

In summary, all code is there but it's in the wrong place and it has the semantics of geofencing and location updates mixed up. Why?

The correct order IMO is:

1. start a service to request location updates via LocationManager. This needs to in the context of a service so that the LocationListener is alive when the callback happens.
2. add geofencing and register the IntentService to handle geofencing events.